src/Alert/Alert.js

```diff
@@ -104,18 +119,60 @@ export const Alert = ({ classes = {}, children, ...props }) => {
         );
         break;
 
+      case 'errorSummary':
+        levers.wrapper =
+          'su-bg-digital-red su-bg-opacity-20 su-text-digital-red';
+        levers.body = redText;
+        levers.dismiss = 'red';
+        levers.dismissIcon = 'x';
+        levers.container = 'su-flex-row su-flex-nowrap';
+        levers.dismissWrapper = 'su-w-auto su-mt-0 su-rs-ml-0';
+        levers.headerWrapper = 'su-w-auto su-mt-0 su-mr-02em';
```

Comment:
```suggestion
        levers.headerWrapper = 'su-w-auto su-mt-0 su-mr-01em';
```
01 looks a bit closer